index.html

```diff
@@ -120,14 +133,30 @@ <h1 id="result-text">Result:</h1>
           id="new-currency-rate-input"
           required
         />
-        <input
+        <!-- <input
           type="text"
           placeholder="insert new currency symbol"
           id="new-currency-symbol-input"
-        />
+        /> -->
       </div>
-    </div>
 
+      <div id="grid-container">
+        <h1>Rates to other currencies</h1>
+        <div class="currency-grid" id="currencies-grid">
+          <div class="grid-header">Base</div>
+          <div class="grid-header">To</div>
+          <div class="grid-header">Rate</div>
+
+         
+      </div>
+    </div>
+    <ul id="saved-alerts-container"></ul>
+  <ul id="alerts-container"></ul>
```

Comment:
I recommend trying to get into the habit of [formatting](https://code.visualstudio.com/docs/editor/codebasics#_formatting) your code before committing it. That way it will look more professional in code assignments for job interviews and so on :)

Reply:
I have been using prettier, but is it the indentation that is wrong?